goss-testing/suites/ncn-csm-health-validation.yaml

```diff
@@ -0,0 +1,37 @@
+#
+# MIT License
+#
+# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# This test suite is intented to run as part of check of system management 
+# monitoring tools on new environment for validation and fix of CMN issues.
+# This suite must run during CSM health validation PIT node is redeployed.
+gossfile:
+  ../tests/goss-switch-ping-status.yaml: {}
+  ../tests/goss-k8s-lb-status.yaml: {}
+  ../tests/goss-k8s-metallb_configmap-status.yaml: {}
+  ../tests/goss-check-prometheus-module-status.yaml: {}
+  ../tests/goss-check-powerdns-status.yaml: {}
+  ../tests/goss-check-default-gw.yaml:  {}
+  ../tests/goss-check-customer-mgm-address.yaml:  {}
+  ../tests/goss-bond0-ip-validation:  {}
+  ../tests/goss-bgp-established-mellanox.yaml:  {}
```

Comment:
If this runs both the Mellanox and the Aruba test, isn't one of them expected to fail?

Reply:
I don't know why [this comment](https://github.com/Cray-HPE/csm-testing/pull/365/files#r946122061) was marked resolved when I still see the issue in the suite file. If it is not a problem having both Aruba and Mellanox in the suite file, please respond with a comment here explaining why. Please do not mark the issue resolved without either addressing it or explaining why it is not a problem -- otherwise we risk things slipping through the cracks.